src/pages/accounting/BalancesTable.tsx

```diff
@@ -50,29 +52,33 @@ function BalancesTable({ accounting, isLoadingUncashed }: Props): ReactElement |
               <TableCell className={classes.values}>
                 <span
                   style={{
-                    color: values.balance > 0 ? '#32c48d' : '#c9201f',
+                    color: balance > 0 ? '#32c48d' : '#c9201f',
                   }}
                 >
-                  {fromBZZbaseUnit(values.balance).toFixed(7).toLocaleString()}
+                  {fromBZZbaseUnit(balance).toFixed(7).toLocaleString()}
                 </span>{' '}
                 BZZ
               </TableCell>
               <TableCell className={classes.values}>
-                -{fromBZZbaseUnit(values.sent).toFixed(7)} / {fromBZZbaseUnit(values.received).toFixed(7)} BZZ
+                -{fromBZZbaseUnit(sent).toFixed(7)} / {fromBZZbaseUnit(received).toFixed(7)} BZZ
               </TableCell>
               <TableCell className={classes.values}>
                 <span
                   style={{
-                    color: values.balance + values.received - values.sent > 0 ? '#32c48d' : '#c9201f',
+                    color: balance + received - sent > 0 ? '#32c48d' : '#c9201f',
                   }}
                 >
-                  {fromBZZbaseUnit(values.balance + values.received - values.sent).toFixed(7)}
+                  {fromBZZbaseUnit(balance + received - sent).toFixed(7)}
                 </span>{' '}
                 BZZ
               </TableCell>
               <TableCell className={classes.values}>
                 {isLoadingUncashed && 'loading...'}
```

Comment:
I can imagine this expressions expressed as
```tsx
{isLoadingUncashed 
 ? 'loading'
 : uncashedAmount > 0 
    ? <>{fromBZZbaseUnit(uncashedAmount).toFixed(7)} BZZ</>
    : '0 BZZ'
```

Or break them down to smaller components (e.g. Loading/UncashedAmount)

Reply:
Thanks, I decided to go half way as nested ternary operators are tough to read. I agree this should be done as a separate component but I know of few other places where that is the case and would love to address that separately.
```tsx
{isLoadingUncashed && 'loading...'}
{!isLoadingUncashed && <>{uncashedAmount > 0 ? fromBZZbaseUnit(uncashedAmount).toFixed(7) : '0'} BZZ</>}
```